libs/pmp-web/repository/shell/src/lib/pmp-web-repository-shell-routing.module.ts

```diff
@@ -27,6 +29,7 @@ const routes: Routes = [
   {
     path: 'repositories',
     component: RepositoriesComponent,
+    canActivateChild: [AuthPublicGuard],
```

Comment:
I think this is possible to add a wrapper for all these routes.
`
{
  path: '',
  canActivate: [AuthPublicGuard],
  children: [
    ... user, repositories
  ]
}`

And i think that canActivate would be better that canActivateChild